roles/mongodb/meta/main.yml

```diff
@@ -16,3 +16,6 @@ galaxy_info:
         - 7
   categories:
     - system
+dependencies:
+  - role: epel
+    become: yes
```

Comment:
Using `become: yes` when including anything external is usually a bad sign.

The good practice is to `become: yes` for every task in a role where it's 100% needed.
Eg. we do privilege escalation per task (only when command is not executed without sudo), not per role.